src/lib/holocene/input/chip-input.svelte

```diff
@@ -18,12 +18,17 @@
   export let hintText = '';
   export let validator: (value: string) => boolean = () => true;
   export let removeChipButtonLabel: string | ((chipValue: string) => string);
+  export let unroundRight = false;
+  export let unroundLeft = false;
+  export let external = false;
```

Comment:
> Could we keep the tags here in the input (like start workflow?)

@Alex-Tideman we explored keeping the chips in the input, but after talking with design (cc @tigernaut) we opted for this `external` chip option to make sure this works with the existing filter (otherwise there are some questions r.e. the border on the input and responsiveness).

Reply:
Can we add a comment explaining what `external` is?